src/uu/kill/src/kill.rs

```diff
@@ -134,15 +134,9 @@ fn handle_obsolete(args: &mut Vec<String>) -> Option<usize> {
 }
 
 fn table() {
-    let name_width = ALL_SIGNALS.iter().map(|n| n.len()).max().unwrap();
-
     for (idx, signal) in ALL_SIGNALS.iter().enumerate() {
-        print!("{0: >#2} {1: <#2$}", idx, signal, name_width + 2);
-        if (idx + 1) % 7 == 0 {
-            println!();
-        }
+        println!("{0: >#2} {1}", idx + 1, signal);
```

Comment:
I think the `+ 1` is incorrect, it makes the signal numbers to be off by one (for example, `HUP` has a signal number of `1`, see https://www.gnu.org/software/coreutils/manual/html_node/Signal-specifications.html):
```
$ cargo run --features=unix kill -t
 1 EXIT
 2 HUP
 3 INT
 4 QUIT
[...]
```
As GNU `kill` doesn't show the `EXIT` signal, you could simply skip the first element with the iterator.

Reply:
Pushed an update to skip the `EXIT` signal.

Reply:
Sorry, my advice was not 100% correct. While working on https://github.com/uutils/coreutils/pull/6222 I learned that not all systems seem to have an `EXIT` signal, and so I replaced `skip` with a `filter`.